contracts/eth/nearbridge/contracts/NearBridge.sol

```diff
@@ -242,7 +242,7 @@ contract NearBridge is INearBridge, AdminControlled {
             untrustedNextHash = nearBlock.next_hash;
 
             uint256 signatureSet = 0;
-            for ((uint i, uint cnt) = (0, thisEpoch.numBPs); i < cnt; i++) {
+            for ((uint i, uint cnt) = (0, thisEpoch.numBPs); i < cnt; ++i) {
```

Comment:
There's not that much difference between pre- and post- increments here because usually, the compiler is smart enough to optimize this in `for` loops.

Reply:
I agree there is not much difference between them but you are using both pre- and post- increments throughout your code. better to just stick with one or the other for better consistency.